plugin/storage/cassandra/schema/v004-go-tmpl-test.cql.tmpl

```diff
@@ -0,0 +1,84 @@
+-- There are total 9 queries here
```

Comment:
what is "test"?

Reply:
[Improved test and changed name in recent commit] The test is to check if the extracted query strings are same as the expected query strings.